py/setup.py

```diff
@@ -27,7 +27,7 @@
 setup_args = {
     'cmdclass': {'install': install},
     'name': 'selenium',
-    'version': "4.4.2",
+    'version': "4.4.3",
```

Comment:
LGTM 👍 , we should probably pin `certifi` inside the `install_requires` in here, tho I don't think this file is fully used, good to keep in sync